internal/plugins/golang/v2/plugin.go

```diff
@@ -45,9 +44,15 @@ var (
 // add operator-framework features to the base kubebuilder Go scaffold and CLI.
 type Plugin struct{}
 
-func (Plugin) Name() string                       { return (kbgov2.Plugin{}).Name() }
-func (Plugin) Version() plugin.Version            { return (kbgov2.Plugin{}).Version() }
-func (Plugin) SupportedProjectVersions() []string { return (kbgov2.Plugin{}).SupportedProjectVersions() }
+func (Plugin) Name() string {
```

Comment:
I assume these gofmt changes are a result of using go v1.14. These changes should be made separately, and we should consider bumping the SDK's go version to v1.14.

Reply:
I switched this back to the original format.  Not sure but it must have been my local dev setup.

Reply:
If you are using the latest version of the lint and then., run `make lint-fix` it will fix the layout in the way that works for **both** versions. I have been sofer resting the changes in this file. Could we please approve https://github.com/operator-framework/operator-sdk/pull/3270 ?

c/c @jmrodri